d3rlpy/cli.py

```diff
@@ -65,6 +65,9 @@ def stats(path: str) -> None:
 @click.option("--ylim", nargs=2, type=float, help="limit on y-axis (tuple).")
 @click.option("--title", help="title of the plot.")
 @click.option("--ylabel", default="value", help="label on y-axis.")
+@click.option(
+    "--save", default=None, nargs=1, help="save to named png or pdf file."
```

Comment:
Could you change this line since we don't need `default` and `nargs` arguments?
```
@click.option("--save", help="flag to save the plot as an image.")
```